[2022-10-13 12:17:12,326] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt.dbt_deps scheduled__2022-10-13T00:03:00+00:00 [queued]>
[2022-10-13 12:17:12,331] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt.dbt_deps scheduled__2022-10-13T00:03:00+00:00 [queued]>
[2022-10-13 12:17:12,331] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-13 12:17:12,331] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-13 12:17:12,331] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-13 12:17:12,348] {taskinstance.py:1377} INFO - Executing <Task(DbtDepsOperator): dbt_deps> on 2022-10-13 00:03:00+00:00
[2022-10-13 12:17:12,352] {standard_task_runner.py:52} INFO - Started process 236094 to run task
[2022-10-13 12:17:12,358] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dbt', 'dbt_deps', 'scheduled__2022-10-13T00:03:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/dbt_airflow.py', '--cfg-path', '/tmp/tmpxgs1zz88', '--error-file', '/tmp/tmpypfnz6a1']
[2022-10-13 12:17:12,359] {standard_task_runner.py:80} INFO - Job 67: Subtask dbt_deps
[2022-10-13 12:17:12,434] {task_command.py:370} INFO - Running <TaskInstance: dbt.dbt_deps scheduled__2022-10-13T00:03:00+00:00 [running]> on host ubuntu
[2022-10-13 12:17:12,499] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dbt
AIRFLOW_CTX_TASK_ID=dbt_deps
AIRFLOW_CTX_EXECUTION_DATE=2022-10-13T00:03:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-13T00:03:00+00:00
[2022-10-13 12:17:12,500] {dbt_hook.py:117} INFO - dbt deps
[2022-10-13 12:17:12,510] {dbt_hook.py:126} INFO - Output:
[2022-10-13 12:17:15,806] {dbt_hook.py:130} INFO - 19:17:15  Running with dbt=1.1.0
[2022-10-13 12:17:15,818] {dbt_hook.py:130} INFO - 19:17:15  Warning: No packages were found in packages.yml
[2022-10-13 12:17:17,226] {dbt_hook.py:132} INFO - Command exited with return code 0
[2022-10-13 12:17:17,239] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dbt, task_id=dbt_deps, execution_date=20221013T000300, start_date=20221013T191712, end_date=20221013T191717
[2022-10-13 12:17:17,285] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-13 12:17:17,325] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
